chore: update html sitemap margin and font size on mobile

sethfeingold opened the pull request.

tihuan reviewed (approved):
Quick question, is `sitemap.tsx` the same as `html-sitemap.tsx`? If so, maybe worth removing `html-sitemap.tsx`? Thank you!

Otherwise LGTM!

sethfeingold commented:
> 

good catch! Thank you! I had deleted that/renamed it locally - thought it would've updated the remote branch but apparently kept both files around. Now deleted! 😄
